update_security_access_file/README.rst

```diff
@@ -0,0 +1,53 @@
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Update Security Access File
+===========================
+
+This module was written for **developpers** to easily generate and update
+model security files.
+
+It search all the models defined by the module (except transient and abstract)
+and then produce a security file for those models.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* go to modules list view.
+* choose a module and click on /!\\ image.
```

Comment:
Please insert the icon image here. Maybe another icon would be more aproppiated for this option.